src/test/java/rotateit/controller/LabelWebhookControllerTest.java

```diff
@@ -0,0 +1,51 @@
+package rotateit.controller;
+
+import com.google.common.io.Resources;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.io.IOException;
+import java.net.URL;
+
+import static com.google.common.base.Charsets.UTF_8;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.standaloneSetup;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration
+@WebAppConfiguration
+public class LabelWebhookControllerTest {
+    private MockMvc helper;
+
+    @InjectMocks
+    private LabelWebhookController labelWebhookController;
```

Comment:
Ok, do it like this:
You don't need context, create controller object in setup and remove `@InjectMocks`
`labelWebhookController = new LabelWebhookController();`
You also don't need `@ContextConfiguration` so:
```
@RunWith(SpringJUnit4ClassRunner.class)
@WebAppConfiguration
public class LabelWebhookControllerTest {
    private MockMvc helper;

    private LabelWebhookController labelWebhookController;

    @Before
    public void setUp() {
        labelWebhookController = new LabelWebhookController();
        helper = standaloneSetup(labelWebhookController).build();
    }
```